theme/static/css/style.css

```diff
@@ -106,6 +106,21 @@ main {
 }
 main #content { flex: 1; }
 
+main #funders {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  align-items: center;
+  max-height: 100px;
+}
+main #cbbs, #efre {
```

Comment:
Nitpick: I think you mean the following:

``main #cbbs, main #efre {``

Reply:
Agree.